docs/monitoring/autodiscovery.md

```diff
@@ -0,0 +1,60 @@
+
+
+Autodiscovered services can be monitored in two ways:
+- **Manually** - in case of bare-metal/virtualMachine setups, you can use classic Agent installation instructions known from the past. In container environments you can just add MONITORING_TOKEN as environment variable to your container and Sematext Agent will automatically match it to the type of discovered service and set up monitoring agent container specifically for that service. In some cases you'll also have to provide credentials (like MySQL username/password) as environment variable or (in case of k8s) as a secret. Again, just follow the classic monitoring installation instructions in UI to set this up.
+- **Automatically** - this new feature is available via UI when clicking on discovered service in Discovery view. In many cases Sematext Agent will be able to set up monitoring agent without requiring any changes from you and without restart of service you wish to monitor. As services appear over time, Sematext Agent automatically ensures new monitoring agents are created when the rule specified in the UI defines so. For each discovered service type you can enable or disable automatic monitoring at any point. For example, if Sematext Agent discovered your Elasticsearch cluster and you see it in Discovery UI, you can easily start its monitoring by a single click in the UI, no actions are required from you on Elasticsearch nodes. Similarly, if you decide to stop monitoring the cluster, single click in the UI will do the trick again. Automatic autodiscovery monitoring is supported both in bare-metal/virtualMachine and container environments. 
+
+<img class="content-modal-image" alt="Enabling Automatic Autodiscovery Monitoring" src="../images/monitoring/automatic-autodiscovery-monitoring.png" title="Enabling Automatic Autodiscovery Monitoring">
+
+### How to start the Autodiscovery?
+
+Simply visit Discovery option in the menu on the left side. You will be presented with instructions to install Sematext Agent. On each machine (or k8s/swarm cluster) Sematext Agent
+is installed, it will instantly start discovering services that can be monitored. Those services will be displayed in Autodiscovery table from where you can click on it to enable automatic monitoring. In most cases Sematext Agent knows how to set up the monitor without requiring any additional actions from you, which means the metrics will start appearing few seconds after you enable automatic monitoring. In other cases you may have to provide credentials so monitoring agents can connect to the service you wish to monitor (in any case, Autodiscovery UI will tell you exactly what to do).
+
+### How does Sematext Autodiscovery Monitoring work in bare-metal/virtualMachine environments?
+
+It is enough to install sematext-agent rpm/deb package and set up one of your Infra App tokens (just follow UI instructions for this). This will start Sematext Agent which will
```

Comment:
your Infra App tokens (just follow UI instructions for this) -> your Infra App tokens by following UI instructions.